build-gluster-org/jobs/lcov.yml

```diff
@@ -28,14 +28,14 @@
         link-to-last-build: true
     - post-tasks:
         - matches:
-            - log-text: Build was aborted
+            - log-text: Building remotely
           script: |
-              sudo reboot
+              sudo chown -R jenkins:jenkins $WORKSPACE
     - post-tasks:
         - matches:
-            - log-text: Building remotely
+            - log-text: Build was aborted
           script: |
-              sudo chown -R jenkins:jenkins $WORKSPACE
+              sudo reboot
```

Comment:
Can you call out this `scripts/regression-abort.sh` instead.

Reply:
The script regression-abort.sh includes steps to archive logs and upload it to logs server, should we do that with line-coverage?

Reply:
Yes! We should upload the logs as well.

Reply:
Oh! Okay.

Reply:
Okay.